<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>steppermotor</class>
 <widget class="QMainWindow" name="steppermotor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StepperMotor control</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>961</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Motorcontrol">
     <attribute name="title">
      <string>Motor control</string>
     </attribute>
     <widget class="QLabel" name="IntervallLabel">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>100</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add measurement interval:</string>
      </property>
     </widget>
     <widget class="QLabel" name="PossibleWavelenth">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Possible wavelenth: 120 nm to 300 nm</string>
      </property>
     </widget>
     <widget class="QLabel" name="ReadingText">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>390</y>
        <width>471</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="StepIntervallSelector">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>240</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <double>0.500000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
      <property name="value">
       <double>1.000000000000000</double>
      </property>
     </widget>
     <widget class="QPushButton" name="UploadButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>250</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Upload</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AddIntervallButton">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>290</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLabel" name="lambdaAddLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>131</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add single position [nm]:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="StartIntervallSelector">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>135</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <double>120.000000000000000</double>
      </property>
      <property name="maximum">
       <double>300.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="EndIntervallSelector">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>190</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <double>120.000000000000000</double>
      </property>
      <property name="maximum">
       <double>300.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
     </widget>
     <widget class="QListWidget" name="ListWavelength">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>30</y>
        <width>261</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="StartIntervallLabel">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>140</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Start [nm]:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="SerialPortBox">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="EndIntervallLabel">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>190</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>End [nm]:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ResetButton">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>250</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QLabel" name="SerialPortLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>141</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Selected Serial Port:</string>
      </property>
     </widget>
     <widget class="QLabel" name="ReadingHead">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Reading:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="WavelengthSelector">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>171</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <double>120.000000000000000</double>
      </property>
      <property name="maximum">
       <double>300.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="InfoList">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>10</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Saved positions:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AddButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>171</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLabel" name="StepIntervallLabel">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>246</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Step [nm]:</string>
      </property>
     </widget>
     <widget class="QLabel" name="StatusHead">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SetButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QLabel" name="StatusText">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>360</y>
        <width>481</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="StartingPosition">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>290</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Starting Position</string>
      </property>
     </widget>
     <widget class="QLabel" name="StartingPositionLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Go to Starting Position (120 nm):</string>
      </property>
     </widget>
     <widget class="QPushButton" name="StartButton">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>290</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Received_data">
     <attribute name="title">
      <string>Received data</string>
     </attribute>
     <widget class="QPushButton" name="ResetReceivedDataButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>390</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QLabel" name="ReceivedLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Received data:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="ListReceivedData">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>281</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="LivePlot">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>80</y>
        <width>361</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="LivePlotlabel">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>50</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Live Plot:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LivePlot_ResetButton">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>380</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ZoomLivePlot_Button">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>380</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Zoom</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Send_result">
     <attribute name="title">
      <string>Send result</string>
     </attribute>
     <widget class="QLabel" name="EnterEmail_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Email:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Email_Edit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>290</y>
        <width>311</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="SendResult_Button">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>340</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Send result </string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="Preview">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>80</y>
        <width>361</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Preview_label">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>50</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Preview:</string>
      </property>
     </widget>
     <widget class="QLabel" name="DataType_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Select data type:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="DataType1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>112</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Type 1</string>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="DataType2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>112</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Type 2</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="DataType3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>70</y>
        <width>112</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Type 3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>What should be sent?</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="OptionData">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>112</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="OptionPlot">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>200</y>
        <width>112</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Plots</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
